### PR TITLE
Profile Page

### PR DESCRIPTION
**Pull request by Dimavich**

### CODE REVIEW

File: client/src/components/posts/PostCard.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { Link } from 'react-router-dom'
 import styled from 'styled-components'
 import { VerticalDiv } from '../../pages/Landing'
-import { QUERY_ALL_USERS } from '../../utils/queries'
+// import { QUERY_ALL_USERS } from '../../utils/queries'
```
Comment (vasylynash) [inline client/src/components/posts/PostCard.js]: Add semicolons after import statements

File: client/src/components/posts/SearchPost.js [hunk 2]
```diff
@@ -40,6 +38,13 @@ export const SearchBtn = styled.button`
     border-radius: 3px;
     color: white;
     padding: 6px 20px;
+    transition: ease-in-out 0.3s;
+
+    &:hover {
+        background-color: #FF8985;
+        color: #707070;
+        cursor: pointer;
```
Comment (vasylynash) [inline client/src/components/posts/SearchPost.js:46]: Add semicolons after import statements

File: client/src/components/profile/Groups.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
```
Comment (vasylynash) [inline client/src/components/profile/Groups.js:1]: Add semicolons

File: client/src/components/profile/Groups.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+function Groups() {
+    return (
+        <div>
+            Currently unavailable
+        </div>
+    )
+}
+
```
Comment (vasylynash) [inline client/src/components/profile/Groups.js:10]: Add semicolons

File: client/src/components/profile/Index.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
```
Comment (vasylynash) [inline client/src/components/profile/Index.js]: Add semicolons after import statements

File: client/src/components/profile/Index.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+            break
+        case 'security':
+            component = <Security/>
+            break
+            default:
+                component = <Info/>
+    }
+    return component;
+}
+
```
Comment (vasylynash) [inline client/src/components/profile/Index.js:20]: Add semicolons after export

File: client/src/components/profile/Info.js [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+                    multiline
+                    rows={2}
+                    />
+                    <LoginBtn type='submit'>Update</LoginBtn>
+                </VerticalDiv>
+            </form>
+        </VerticalDiv>
+     );
+}
+export default Info
```
Comment (vasylynash) [inline client/src/components/profile/Info.js]: Add semicolons

File: client/src/components/profile/Security.js [hunk 8]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
```
Comment (vasylynash) [inline client/src/components/profile/Security.js]: Add semicolons after import statements

File: client/src/components/profile/Security.js [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+                    <h2 style={{fontSize:'20px', marginTop:'1rem'}}>Change password</h2>
+                    <TextField id="password" label="Password" color='primary' type='password' />
+                    <TextField id="confirmPassword" label="Confirm password" color='primary' type='password'/>
+                    <LoginBtn type='submit'>Change</LoginBtn>
+                </VerticalDiv>
+            </form>
+        </>
+    )
+}
+
```
Comment (vasylynash) [inline client/src/components/profile/Security.js:31]: Add semicolons

**Review by vasylynash — CHANGES_REQUESTED** [review]

**Review by Zoneam — APPROVED** [review]

**Review by melinamboedecker — APPROVED** [review]

**Review by vasylynash — APPROVED** [review]

**Pull request merged**